Sources/IO/Geometry/STLReader/index.js

```diff
@@ -110,6 +135,49 @@ function vtkSTLReader(publicAPI, model) {
       cellDataValues[faceIdx] = dataView.getUint16(offset + 48, true);
     }
 
+    // Rotate points
+    const orientationField = 'SPACE';
+    if (orientationField in header && header[orientationField] !== 'LPS') {
+      const XYZ = header[orientationField];
+      const mat4 = [...new Array(16)].map((x) => 0);
```

Comment:
new Float32Array(16); will already be filled with 0.

Reply:
@jourdain when mat4 is a `Float32Array`, `Array.isArray(mat4x4)` returns `false` in MatrixBuilder (but no warning or error message)
https://github.com/Kitware/vtk-js/commit/d1fb6679af8edfeffd4644c46e3246e09ee2abe8#diff-915eb56ed796b4179f838589e323b110R111

Which part should I modify? `MatrixBuilder.setMatrix` or this?